fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -309,9 +346,21 @@ public Table getTable(String dbName, String tblName) {
             table.setComment(icebergTable.properties().getOrDefault(COMMENT, ""));
             tables.put(identifier, table);
             return table;
-        } catch (StarRocksConnectorException | NoSuchTableException e) {
+        } catch (StarRocksConnectorException e) {
             LOG.error("Failed to get iceberg table {}", identifier, e);
             return null;
+        } catch (NoSuchTableException e) {
+            return getView(dbName, tblName);
+        }
+    }
+
+    public Table getView(String dbName, String viewName) {
+        try {
+            View icebergView = icebergCatalog.getView(dbName, viewName);
+            return IcebergApiConverter.toView(catalogName, dbName, icebergView);
+        } catch (Exception e) {
```

Comment:
catch (StarRocksConnectorException | NoSuchTableException e) ?

Reply:
if we failed to getTable, we need to return null. So the `getView` is consistent with `getTable`.